src/remote/integration.ts

```diff
@@ -50,42 +16,15 @@ function remoteExtensionsInstalled(): boolean {
 export async function addRemoteIntegration(context: vscode.ExtensionContext) {
   State.extensionContext = context;
 
-  registerRemoteIntegrationCommands();
-
   const remoteExtensions = remoteExtensionsInstalled();
   await vscode.commands.executeCommand('setContext', 'peacock:remote', remoteExtensions);
```

Comment:
The context `peacock:remote` was only used by the remote commands, since the remote commands got removed it is no longer necessary to set this context. As a consequence the function `remoteExtensionsInstalled` is also no longer needed.

Reply:
thanks, done